buildSrc/src/main/kotlin/ProtobufConfiguration.kt

```diff
@@ -38,9 +38,10 @@ object ProtobufConfiguration {
         val protoFiles = mutableSetOf<String>()
         rootDir.walk().forEach {
             if (it.isFile && it.extension == "proto") {
-                protoFiles.add(it.relativeTo(rootDir).path)
+                val path=it.relativeTo(rootDir).path.replace("\\", "/")
```

Comment:
直接用 File.invariantSeparatorsPath 貌似更好
```suggestion
                val path = it.relativeTo(rootDir).invariantSeparatorsPath
```

Reply:
这个确实可以，刚刚一时忘了 API 名字